webrtc.html

```diff
@@ -9085,11 +9075,13 @@ <h3><dfn>RTCDataChannel</dfn></h3>
           </ul>
         </li>
         <li>
-          <p>Fire an event named <code><a>message</a></code> at
-          <var>channel</var> with the <code>origin</code> attribute initialized
+          <p><a>Fire an event</a> named <code><a>message</a></code> using the 
+          <code title="event-datachannel-message"><a>message</a></code> interface
+          with the <code>origin</code> attribute initialized
           to the origin of the document that created the <var>channel</var>'s
           associated <a>RTCPeerConnection</a>, and the <code>data</code>
```

Comment:
This seems too vague. You need to define that RTCPeerConnection holds a document in a slot somewhere and use that concept here. And define how the slot gets filled when creating RTCPeerConnection.

Reply:
Agreed, I opened #1969. However, goes beyond the scope of this PR.